verl/trainer/config/gsm8k.yaml

```diff
@@ -0,0 +1,188 @@
+data:
```

Comment:
It might be preferable to maintain `ppo_trainer.yaml` with added fields instead of creating two separate dataset-specific YAML configurations in `trainer/config`. These two YAML configurations can be utilized in examples, and the `--config-path` parameter can be assigned for use. For simpler cases, modifying some arguments should suffice.